nodejs/week1/src/backend/index.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
```

Comment:
Great job!

I think there's one part missing. You need to make your server more modular by breaking down the routes into files as detailed here:

https://github.com/Vintharas/node.js-1/blob/master/week1/homework.md#modular-routes